README.md

```diff
@@ -1,33 +1,35 @@
 # Infinity Works - Full Stack Tech Test: Python
 
-Python version of the Infinity Works technical test. It uses the `flask`_ microframework.
-
-.. _flask: http://flask.pocoo.org
+Python version of the Infinity Works technical test. It uses the [Flask](https://flask.palletsprojects.com/) microframework.
 
 ![Preview of Frontend](preview.png)
 
 ## Prerequisites
 
-* Python 3
+* [Python 3.7 or newer](https://www.python.org/)
 * Clone this repo
 * Access to the internet
 * Suitable development environment
 
 ## Getting Started
 
 * Create and activate a new virtual environment:
+    ```bash
+    python3 -m venv .venv
+    source .venv/bin/activate
+    ```
 
-    $ python3 -m venv venv
-    $ . venv/bin/activate
-
-* Install dependencies::
-
-    $ pip install -e .
+* Install dependencies:
+    ```bash
+    pip install -r requirements.txt
+    ```
 
 * Start the app:
-    $ python app.py
+    ```bash
+    flask run
+    ```
 
-* Visit http://localhost:5000 to receive the application.
+* Visit [`http://127.0.0.1:5000/`](http://127.0.0.1:5000/) to view the application.
```

Comment:
Can still point to local? 127.0.0.1 is local host? Seems an unnecessary a change.

Reply:
Just reused Flask's output but happy to change.

![image](https://user-images.githubusercontent.com/1130363/150546278-1b3b8c8b-6a99-47fc-8f95-d5897c50f2a4.png)